content/en/examples/configmap/immutable-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+data:
+  season: "summer"
+kind: ConfigMap
+immutable: true
+metadata:
+  name: season
```

Comment:
I think this isn't an ideal example of immutable data. Can we think of a setting that a human coder would reasonably think “yes, I can see why that wouldn't change very often”?

How about:
```yaml
data:
  cloud_provider_region: antarctica-1
```

or
```yaml
data:
  operating_system: Linux
```

Reply:
Thanks. I have implemented the example with the company names. It is available in the latest commit.